core/src/main/java/io/questdb/cairo/pool/SqlCompilerPool.java

```diff
@@ -9,13 +9,10 @@
 import io.questdb.std.Rnd;
 
 public final class SqlCompilerPool extends AbstractMultiTenantPool<SqlCompilerPool.C> {
-    private static final long INACTIVE_COMPILER_TTL_MILLIS = 5 * 60 * 1000L; // 5 minutes
-    // todo: consider making the constants configurable
-    private static final int MAX_POOL_SEGMENTS = 64;
     // todo: are you sure there are no side effects here? for example the dirName? at very least it's being logged
```

Comment:
@jerrinot do you recall this TODO? Can we remove it now?

Reply:
indeed the TODO is not very clear. It meant to say this:
```
    private static final TableToken[] TOKENS = {
            new TableToken("blue", "/compilers/blue/", 0, false, false, false),
            new TableToken("red", "/compilers/red/", 0, false, false, false),
            new TableToken("green", "/compilers/green/", 0, false, false, false)
    };
```
The `"/compilers/red/"` are fake directories. As the pool impl is/was meant to be used for pooling table readers/writers. So `/compilers/<color>` is bit of a hack and the TODO is just me musing whether this could possible cause some confusion, e.g. when analysing logs, etc.

Reply:
Sounds like this todo should be merged into the below note. WDYT?

Reply:
turned that todo to a comment in 90efe4b82cfd97f029bd391df4180704aea7575d

Reply:
thanks @glasstiger, this is great!